src/neo/Trie/MPT/MPTNode/ShortNode.cs

```diff
@@ -0,0 +1,48 @@
+using System.IO;
+using Neo.Cryptography;
+using Neo.IO;
+
+namespace Neo.Trie.MPT
+{
+    public class ShortNode : MPTNode
+    {
+        public byte[] Key;
+
+        public MPTNode Next;
+
+        public new int Size => Key.Length + Next.Size;
```

Comment:
I thinck we should override the Size attribute in MPTNode

Reply:
👌